### ISSUE TITLE
Minor UI inconsistency in extension button

### ISSUE THREAD

**Issue by hrxn** [gorhill/uBlock#374]
Just noticed this behaviour, it's probably UI only, nothing else wrong or broken so far...
1. At the start of Chrome, this happens (first launch, not running before etc.)
   ![ublocko-firstlaunch](https://cloud.githubusercontent.com/assets/12064918/8253962/c19b4558-1694-11e5-851f-f14dfac17c7d.png)
2. Open a new tab. Or a new window, doesn't matter. And then this...
   ![ublocko-newtab](https://cloud.githubusercontent.com/assets/12064918/8253974/d86aa116-1694-11e5-8bb9-cd07f4e3f317.png)
## 

uBlock Origin 0.9.9.1
Chrome Stable Version 43.0.2357.124 m (64-bit)
Windows 7 SP1 x64

**Comment by gorhill** [comment]
I need to know the URL in the address bar for the first picture.

**Comment by Hrxn** [comment]
Uh, address?

That page is the standard Google start page presented upon launch, nothing changed. 
Basically a fresh Chrome install.

In chrome://settings/ the option selected under 'On startup' is 'Open the New Tab page

**Comment by gorhill** [comment]
> In chrome://settings/ the option selected under 'On startup' is 'Open the New Tab' page

I did set as indicated: unable to reproduce using Chromium 43.

**Comment by Hrxn** [comment]
For the record, the start page looks like this:

![chromestartpage](https://cloud.githubusercontent.com/assets/12064918/8290558/10335908-1924-11e5-9f42-2e678bbda65f.png)

The address bar is empty (only the blinking cursor, of course).


Could be an OS specific issue (Windows). I tested it with different profiles, but result is the same.

**Comment by Hrxn** [comment]
So, the new profile with freshly installed uBlock:

![Uploading chromenewprofile.png…]()

**Comment by gorhill** [comment]
Digging a bit more, here is what I found:

When I launch Chromium, the internal URL (not shown in the address bar) is `chrome-newtab`. When I create a new tab, the internal URL (not shown) is `www.google.ca` (on my side). So although to us the two _New Tab_ page look just like the same, they have different internal URLs.

The `chrome-scheme` is whitelisted by default out of the box (I mine was not default setting), hence the _New Tab_ used at launch will have uBlock disabled, while the _New Tab_ created from adding a tab will not be whitelisted by default.

There is nothing really I can do, Chromium internally displays two different pages with different URLs, though to us they look just like the same.

**Comment by Hrxn** [comment]
Woops, looks like I accidentally closed the issue. This was not intended. Stay away from computer when drunk, I guess.

I added the missing screenshot to my last comment, btw. Just as a proof ;-)

Two different pages internally, interesting.

So I guess it's not related to the OS, looks more like a Chrome vs. Chromium issue at the moment. If I understood that correctly.

I will do some more testing later, and see if I am able to reproduce the issue somehow.

BTW, thanks for your great work, Raymond!

**Comment by gorhill** [comment]
See <https://github.com/gorhill/uBlock/issues/374#issuecomment-114268155> above, I explain why this happens, and there is no fix, it's a browser quirk.

**Comment by Hrxn** [comment]
Okay, got it..

When I'm home again I will try it with Chromium, anyway.

And maybe open an issue here:

https://code.google.com/p/chromium/

**Comment by Hrxn** [comment]
Well, it has been a while, but I did test it with Chromium in the meantime.

I could reproduce the issue described in the initial comment, UI inconsistency is also happening with Chromium.

I tested:

Chromium for Windows x86, 32-Bit, Snapshot Build
Chromium for Windows, both 32- and 64-Bit versions, build from Continuous

On:

Windows 7 Pro x32
Windows 7 HP x64
Windows 7 Pro x64

All with the same results. So I guess both Chrome and Chromium are affected, at least on Windows.

Maybe it is the OS, after all. I mean, you said that you couldn't reproduce it, so at least Chromium on Linux appears to be working...

All this drama with the New Tab Page (NTP) lately...

The Chrome Release Blog full of comments about missing bookmark tiles and app icons.
At least they [acknowledged](http://googlechromereleases.blogspot.com/2015/08/stable-channel-update_20.html?showComment=1440203645414#c3616527433568488382) it.

Like that one guy said.. Google doing tests with the NTP in production environment

;-)
